VaticanBAV/et/BAVet120.xml

```diff
@@ -107,7 +107,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <locus target="#42v"></locus>
 <desc type="Subscription"></desc>
 <q xml:lang="gez">ዝ፡ መጽሐፍ፡ ዘ<placeName ref="LOC1713Bagemd">በጌምድር፡</placeName>
- <placeName ref="LOC3056Farta">ፋርጣ፡</placeName> ዓዲኮ፡ <persName role="owner" ref="PRS12979GabrielFA">ገብርኤል</persName>፡
+    <placeName ref="LOC3056Farta">ፋርጣ፡</placeName> ዓዲኮ፡ <placeName ref="INS1013FAG">ገብርኤል</placeName>፡
```

Comment:
I would only markup `<placeName ref="INS1013FAG">ፋርጣ፡ ዓዲኮ፡ ገብርኤል</placeName>`, the information on warada and qabale is for the settlement section of the INS1013FAG record